Makefile

```diff
@@ -91,7 +91,7 @@ format:
 # Python commands
 collect:
 	@${MAKE} \
-	-e CONTAINER_CMD="bash -lc 'KERNMLOPS_CONFIG_FILE=${KERNMLOPS_CONFIG_FILE} make collect-data'" \
+	-e CONTAINER_CMD="bash -lc 'KERNMLOPS_CONFIG_FILE=${KERNMLOPS_CONFIG_FILE} make collect-raw'" \
```

Comment:
the idea is that this should be controlled by an `overrides.yaml` file, and if one is not specified then it is determined by the `defaults.yaml` file that does do `raw` -- is this not correct?  Or what is the issue you are seeing?

Reply:
I was just frustrated, very late at night sorry for spurious PR.